qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -94,6 +94,13 @@ class TwirlingOptions:
         twirled layer.
 
       Default: "active-accum".
+
+    .. image:: /source_images/twirling_strategy_options.png
+      :alt: Image illustrating the differences among twirling strategies. If "active", only
+            instruction qubits in each layer will be twirled. If "active-circuit" the union of 
```

Comment:
```suggestion
            instruction qubits in each layer will be twirled. If "active-circuit", 
```

Reply:
I think of "union" being synonymous with "all", which is why I removed that word, but if it's important in a meaningful way, that's fine.